pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "tt-umd"
+version = "1.0.0"
+authors = [
+    { name = "Tenstorrent", email = "info@tenstorrent.com" }
```

Comment:
This should be flipped to the maintainers for tt-umd, not an unmonitored e-mail address